rated_list.md

```diff
@@ -37,6 +43,7 @@ SampleId = uint64
 @dataclass
 class NodeRecord:
     node_id: NodeId
+    csc: uint8
```

Comment:
```suggestion
    custody_size_parameter: uint64
```

I want to make this parameter compatible with future upgrades to more samples (e.g. 2d)